extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -27,11 +27,22 @@ namespace minifi {
 namespace processors {
 
 core::Property ListenHTTP::BasePath(
-    core::PropertyBuilder::createProperty("Base Path")->withDescription("Base path for incoming connections")->isRequired(false)->withDefaultValue<std::string>("contentListener")->build());
+    core::PropertyBuilder::createProperty("Base Path")->
```

Comment:
I know we don't quite yet lint extensions ( this will happen soon ) but I'm pretty sure this will need //NOLINT if you intend to keep it this way.

Reply:
That's a good point, thanks!
Checked other processors, it should be the opposite way ( -> operator is at the beginning of new lines) to be linter-compatible, so fixed.